parser/feed/src/main/java/de/danoeh/antennapod/parser/feed/PodcastIndexTranscriptParser.java

```diff
@@ -0,0 +1,191 @@
+package de.danoeh.antennapod.parser.feed;
+
+import android.util.Log;
+
+import org.apache.commons.lang3.StringUtils;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.jsoup.internal.StringUtil;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+
+import de.danoeh.antennapod.model.feed.Transcript;
+import de.danoeh.antennapod.model.feed.TranscriptSegment;
+
+public class PodcastIndexTranscriptParser {
+
+    public static class PodcastIndexTranscriptSrtParser {
```

Comment:
I would do the parser for each format in its own file.

Srt is a generic format, it's not something specific to PodcastIndex. So I would just call it `SrtTranscriptParser` or something similar.

Reply:
done